src/cmd-compress

```diff
@@ -12,10 +12,21 @@ import argparse
 sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
 from cmdlib import run_verbose, write_json, sha256sum_file, rm_allow_noent
 
+# Hardcoded value for compression algorithm for now
+DEFAULT_COMPRESSION_ALGO = 'xz'
+
 parser = argparse.ArgumentParser()
 parser.add_argument("--build", help="Build ID")
+parser.add_argument("--algorithm",
+                    default=DEFAULT_COMPRESSION_ALGO,
+                    help="Compression algorithm to use")
 args = parser.parse_args()
 
+# handle --compression-algorithm
+ext = '.xz'
```

Comment:
Minor: rather than having to keep this in sync with the default, you could do a lookup unconditionally here. E.g. `if-elif` or a dict of `algorithm --> extension`.

Reply:
Updated :+1: